cypress/e2e/shared/explorer.test.ts

```diff
@@ -303,7 +303,7 @@ describe('DataExplorer', () => {
         cy.getByTestID('timerange-popover--dialog').should('have.length', 1)
       })
 
-      it('should error when submitting stop dates that are before start dates', () => {
+      it('should error when submitting stop dates that are before start dates and should error when invalid dates are input', () => {
```

Comment:
consolidating the tests so that we don't have to spin up another test and speed things up